cypress/e2e/smoke/load_safe.cy.js

```diff
@@ -42,26 +42,35 @@ describe('Load Safe tests', () => {
     safe.verifyIncorrectAddressErrorMessage()
     safe.inputNameAndAddress(testSafeName, constants.SEPOLIA_TEST_SAFE_1)
 
-    // Type an invalid address
-    // cy.get('input[name="address"]').clear().type(EOA_ADDRESS)
-    // cy.get('@addressLabel').contains(INVALID_ADDRESS_ERROR_MSG)
-
-    // Type a ENS name
-    // TODO: register a goerli ENS name for the test Safe
-    // cy.get('input[name="address"]').clear().type(SAFE_ENS_NAME)
-    // giving time to the ENS name to be translated
-    // cy.get('input[name="address"]', { timeout: 10000 }).should('have.value', `rin:${SAFE_ENS_NAME_TRANSLATED}`)
-
-    // Uploading a QR code
-    // TODO: fix this
-    // cy.findByTestId('QrCodeIcon').click()
-    // cy.contains('Upload an image').click()
-    // cy.get('[type="file"]').attachFile('../fixtures/goerli_safe_QR.png')
-
-    safe.verifyAddressInputValue()
+    safe.verifyAddressInputValue(constants.SEPOLIA_TEST_SAFE_1)
+    safe.verifyNextButtonStatus('be.enabled')
     safe.clickOnNextBtn()
   })
 
+  it('Verify names cannot have more than 50 chars', () => {
+    safe.inputName(main.generateRandomString(51))
+    safe.verifyNameLengthErrorMessage()
+  })
+
+  it('Verify ENS name is translated to a valid address', () => {
+    safe.inputAddress(constants.ENS_TEST_ETH)
+    safe.verifyAddressInputValue(constants.ETH_ENS_SAFE_ADDRESS_7)
+    safe.verifyNextButtonStatus('be.enabled')
+    safe.clickOnNextBtn()
+  })
+
+  it('Verify a valid QR code is accepted', () => {
```

Comment:
I need some help here. 
If I leave it commented this way then the ScanQRCode function runs just fine. But for some reason if I uncomment all those lines, the function ScanQRCode is ignored, the VerifyAddressInputValue function (that checks if the address in the input is valid) kicks in first and of course fails since the QR code was not uploaded. I don't know why this happens

Reply:
The reason for it is that ETH_ENS_SAFE_ADDRESS_6 does not exist it throws the error as undefined. To fix it add ETH_ENS_SAFE_ADDRESS_6 to constants.js. After this fix everything should work as expected.

Reply:
I wrote the wrong name of the constant. Fixed. Thanks